queries/PublicationCount.rq

```diff
@@ -1,9 +1,8 @@
 PREFIX bibo: <http://purl.org/ontology/bibo/>
 PREFIX dco: <http://info.deepcarbon.net/schema#>
 
-#
 #Counts the total number of publications on the DCO website
-#
+#Stephen Moon, July 2015
 SELECT (concat("Total Publications: ", (COUNT(?publication))) as ?number) WHERE
```

Comment:
Just tested it out, both give the same result (1532 publications).

On Mon, Jul 20, 2015 at 11:16 AM, Patrick West <notifications@github.com>
wrote:

> In queries/PublicationCount.rq
> <https://github.com/tetherless-world/dco-twsparql/pull/2#discussion_r35005599>
> :
>
> >  #Counts the total number of publications on the DCO website
> > -#
> > +#Stephen Moon, July 2015
> >  SELECT (concat("Total Publications: ", (COUNT(?publication))) as ?number) WHERE
>
> Do we need a distinct in front of ?publication
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tetherless-world/dco-twsparql/pull/2/files#r35005599>.
>



-- 
Stephen T. Moon
(860) 480-8751